### DISCUSSION TITLE
Please add more tutorials, especially state management

### DISCUSSION THREAD

**Discussion by jet10000** [nylo-core/nylo#21] [General] [answered]
for newbies

**Comment by agordn52** [comment]
I've been working on a series but it's not finished yet.
I'll ping a notification here when it's up.

Also, if anyone else wants to create videos, that'd be cool.
Docs are live https://nylo.dev/ 🚀

**Reply by agordn52** [reply]
I have scheduled some videos for the 18th of July.

You can subscribe to the official YouTube channel here for updates https://www.youtube.com/channel/UCUbk3oL0H3luYyhq-pnozNg

**Comment by agordn52** [comment, marked as answer]
The YouTube channel is now up and running for Nylo... I hope the videos help 🙂

https://www.youtube.com/channel/UCUbk3oL0H3luYyhq-pnozNg
